### PR TITLE
Added missing alt text

### PR DESCRIPTION
**Pull request by MeeraDi**

### CODE REVIEW

File: iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/userIsolation/index.md [hunk 1]
```diff
@@ -143,9 +143,9 @@ To support FTP publishing for your Web server, you must install the FTP service.
         - Double-click **Administrative Tools**, and then double-click **Internet Information Services (IIS) Manager**.
 2. In the **Connections** pane, expand the server name, expand the **Sites** node, then click the name of the site.
 3. In the site's **Home** pane, double-click **FTP User Isolation**.  
-    [![](index/_static/image10.png)](index/_static/image9.png)
+    ![Screenshot of the I I S Manager window showing the website home. The F T P User Isolation icon is selected.](index/_static/image9.png)
 4. On the **FTP User Isolation** page, under **Isolate users. Restrict users to the following directory:**, select **User name directory (disable global virtual directories)**.  
-    [![](index/_static/image12.png)](index/_static/image11.png)
+    ![Screenshot of the I I S Manager window showing the F T P User Isolation window](index/_static/image11.png)
```
Comment (v-emilypr) [inline iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/userIsolation/index.md:148]: Missing period.

**Comment by MeeraDi** [comment]
@Rick-Anderson @mcleblanc
This PR addresses missing alt text and is ready for final review.

File: iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/userIsolation/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Default FTP User Isolation Settings &lt;userIsolation&gt;"
 author: rick-anderson
-description: "Overview The &lt;userIsolation&gt; element is used to start or restrict FTP clients in specific sections of an FTP site. Depending on the options that are sp..."
+description: "Using the userIsolation element to start or restrict FTP clients in sections of the FTP site."
```
Comment (Rick-Anderson) [inline iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/userIsolation/index.md:4]:
```suggestion
description: "Using the &lt;userIsolation&gt; element to start or restrict FTP clients in sections of the FTP site."
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@MeeraDi  check my commit. You can use either `&lt;userIsolation&gt;` or `\<userIsolation>`

**Pull request merged**